### PR TITLE
Proofreading edits (Ch 14-17)

### PR DESCRIPTION
**Pull request by emilyriederer**
This PR contains proofreading edits. I have done each in a separate to explain what changes were made. Below, I'm logging a few bigger-picture questions and observations. 

**Chapter 14**

- Section 14.9.4 awkwardly split over two pages. We may want to check on this in #210

**Chapter 15**

- Figure 15.2 is another one to keep a close eye on during #210

**Chapter 16**

- It's not entirely clear to me why 16.2 (Read external scripts into a chunk) has some of the plain text fenced in an `md` block instead of just showing it as normal text. This is a trivial point but just mentioning in case it could be confusing to others also.
- Many images in 16.6, so this is another good candidate to double check during #210 

**Chapter 17**

- Table 17.1 is lying nicely now, but this will be something we will have to check again after all edits (in the spirit of #210)
- Shall we reference other books by chapter number or title? For example, here we say "you may read Chapter 15". I think I may have broken from convention here in other places and given the section name instead in case the numbers change in the future. If we care about making this consistent, I can go back through the book and correct instances where I did this a different way.
- Should all level 2 headers be imperative verb phrases ("do X"?) If so, we could change two:
  + RStudio keyboard shortcuts -> Use RStudio keyboard shortcuts
  + Collaborative editing of Rmd documents through Google Drive -> Collaborate on an Rmd in Google Drive

### CODE REVIEW

File: 17-workflow.Rmd [hunk 1]
```diff
@@ -97,7 +101,7 @@ The second argument of `xfun::Rscript_call()` takes a list of arguments to be pa
 
 ## Parameterized reports {#parameterized-reports}
 
-In Section \@ref(rmarkdown-render), we mentioned one way to render a series of reports in a `for`-loop. In fact, `rmarkdown::render()` has an argument named `params` specifically designed for this task. You can parameterize your report through this argument. When you specify parameters for a report, you can use the variable `params` in your report. For example, if you call:
+In Section \@ref(rmarkdown-render), we mentioned that one way to render a series of reports in a `for`-loop. In fact, `rmarkdown::render()` has an argument named `params` specifically designed for this task. You can parameterize your report through this argument. When you specify parameters for a report, you can use the variable `params` in your report. For example, if you call:
```
Comment (yihui) [inline 17-workflow.Rmd:104]: `that` doesn't seem to be necessary here.

Comment (emilyriederer) [reply 17-workflow.Rmd:104]: Ok! Will remove

**Comment by emilyriederer** [comment]
@yihui - Thanks for the replies earlier! I just pushed all changes for Ch 14-17 and am taking a pause. These can either be reviewed now and I can open a new PR for other chapters, or I can just keep pushing to the same branch. Please let me know if either is preferrable!

**Comment by yihui** [comment]
Please send separate PRs for other chapters. I'll review this one soon. Thanks!

**Pull request merged**